app/controllers/actions/JourneyRefiner.scala

```diff
@@ -32,10 +32,11 @@ class JourneyRefiner @Inject()(cacheRepository: CacheRepository)(implicit val ex
   override protected def executionContext: ExecutionContext = exc
 
   private def refiner[A](request: AuthenticatedRequest[A], types: JourneyType*): Future[Either[Result, JourneyRequest[A]]] =
-    cacheRepository.findByProviderId(request.operator.providerId).map(_.map(_.answers)).map {
+    cacheRepository.findByProviderId(request.operator.providerId).map(_.flatMap(_.answers)).map {
       case Some(answers: Answers) if types.isEmpty || types.contains(answers.`type`) =>
         Right(JourneyRequest(answers, request))
-      case _ =>
+      case a =>
+        print(s"foud this " + a)
```

Comment:
If we want to merge this PR please remove this case

Reply:
Oops!